### ISSUE TITLE
Misleading error message with mmap failure

### ISSUE THREAD

**Issue by bjmoor** [sstsimulator/sst-elements#704]
In MemHierarchy's membackend, if the `mmap()` call fails, for example, if you do not have enough RAM on the host system, the resulting error message is that there are conflicting parameters:

```
 sst juno-exercise-001.py
FATAL: 0:--->  MemBackendConvertor, Error - conflicting parameters. 'do_not_back' cannot be true if 'memory_file' is specified.  memory_file =
SST Fatal Backtrace Information:
```

MemHierarchy should have a more intelligent error message in this case.

**Comment by nmhamster** [comment]
Are these fixed in #708?

**Comment by gvoskuilen** [comment]
Fixed in #708
